### ISSUE TITLE
Journals Missing UI Covers

### ISSUE THREAD

**Issue by patmauro** [H-uru/moul-assets#211]
The following journals currently break Uru design language due to missing covers in the UI, which seems to be due to cover definitions missing from their LOC files. In every case, a cover texture does exist for this book already, but was simply not linked to the UI object - in theory, each of these cases should therefore be fairly trivial to fix:

- **Serene Journal, Serene**
- **Tiam Journal 1, Tiam**
- **Tiam Journal 2, Tiam**
- **Tiam Journal 3, Tiam**
- **Elonin Flower Journal, Elonin**
- **Elonin Main Journal, Elonin**
- **Rebus Journal, Chiso Preniv**
- **Dulcamara Journal, Chiso Preniv**
- **Carl Palmer notebook on Kelsei's Desk, Chiso Preniv**

_(Not including the Kalamee journals or Naybree science log here, as these are special cases that don't really fall under the umbrella of the particular issue I am describing here.)_

The behavior of all books objects in Uru needs to be consistent across the board, otherwise they risk breaking the gameplay & design language established by the base game, which in turn breaks immersion.

**Comment by DoobesURU** [comment]
Huh. Wonder how all that slipped past testing. I'll have a look at the PRPs and LOC files and see if I can fix all that.

**Comment by patmauro** [comment]
The problem, as I see it, is that "auto open" behavior is _itself_ not consistent with established Uru behavior - either way, this is going to read as a mistake rather than an intentional design choice. So, I would strongly encourage the writers of each of these journals to make the needed adjustments to bring these journals "up to compliance" as it were, with properly defined covers.

Will make my case with the creators in question, and see if we can't convince folks to make changes bringing these "up to compliance" as it were.

**Comment by patmauro** [comment]
All creators have approved these changes; many of the fixes are already live on OU's m-a test shard. Once all the relevant PRs are submitted to h-uru/moul-assets, this issue can be closed.

**Comment by patmauro** [comment]
Just re-reviewed; as of the 2023Q1 Update all of these have been fixed on MOULa and several have been fixed here in H'uru/moul-assets. The only remaining holdouts on H'uru are due to the following:
- PR 212 (still open) is still missing the fix for this
- PR 215 was missing a few last-minute journal adjustments in Chiso
- Elonin & Bahvanter have not yet been accepted to H'uru (once they are, they will presumably have the needed fix)

**Comment by Hoikas** [comment]
#212 is currently broken as submitted and is awaiting revisions.

**Comment by patmauro** [comment]
Once (1) Chiso gets a supplemental update that adds those last 2 covers and (2) #212 is completed, then I think we are good to close this ticket - since Elonin & Bahvahnter are not part of this repo yet, they are _technically_ out of scope here. Instead, I'll add a reminder on their corresponding PRs to account for this.

**Comment by patmauro** [comment]
@DoobesURU - do you plan to drop a PR for your 2023Q3 adjustments to Chiso with the shelf, etc? If that includes the Dulcamara & Carl Palmer journal fixes as well, then this issue can probably be marked "solved."

**Comment by patmauro** [comment]
I'll have to double-check Chiso once the ongoing "Purge" is done, but as long as Serene & any RP stuff in Chiso is removed, that should hypothetically close this ticket. Will re-check and advise.

**Comment by patmauro** [comment]
List updated to reflect the fan Age "purge." A PR updating Chiso so it is H'uru compliant (separating fan journals out to a different PRP not included in the package prepared for H'uru) will address the final two items and will resolve this Issue. (We can assess this once a new blend for Chiso is obtained.)
